### ISSUE TITLE
[Feature Request] Pref. Setting Controlling Replace-in-Files Confirmation

### ISSUE THREAD

**Issue by alankilborn** [notepad-plus-plus/notepad-plus-plus#12423]
Can we get a setting controlling _Replace in Files_ confirmation box appearance?

We already have this for _Replace All in All Opened Documents_, since v7.9:

![image](https://user-images.githubusercontent.com/77065706/198828178-5ba94d0d-3118-472f-aff6-17cd5bd7704d.png)

And a new setting would fit nicely below that one:  "Confirm Replace in Files"
And, like the existing setting, this one would be a checkbox that is checked by default.

To summarize, I'd like to be able to control whether or not this appears when doing a replacement in files operation:

![image](https://user-images.githubusercontent.com/77065706/198828148-7de11120-0313-441c-9bff-a6503c674039.png)

Request based on existing functionality in v8.4.6.
